doc/howto/instances_manage.md

```diff
@@ -36,6 +36,13 @@ Enter the following command to start an instance:
 
 You will get an error if the instance does not exist or if it is running already.
 
+To immediately attach to the console when starting, pass the `--console` flag.
```

Comment:
You need to do just `--console` for containers or VMs to get the text console, and then for VMs you can optionally do `--console=vga` if you need the graphical console.

Reply:
Right, the command is just one example. ;)
But changed it to just `--console` now since I have both in the other page.